notebooks/Emukit-task-spread-of-disease-with-seir-model.ipynb

```diff
@@ -0,0 +1,908 @@
+    "meanmax = seir.MeanMaxInfectionGillespie(seir_gil, num_gil, t_end, alpha_prior)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Task 1: Height of peak of infection\n",
+    "\n",
+    "The height of the peak is the expected maximal number of simultaneously infected individuals \n",
+    "\n",
+    "$$\n",
+    "\\mathbb{E}_a\\left[\\max_t N_I(t, \\alpha)\\right] =\\int_{\\Omega}\\max_t N_I(t, \\alpha)p(\\alpha)\\mathrm{d}\\alpha.\n",
+    "$$\n",
+    "\n",
+    "We use the Bayesian quadrature package of Emukit to emulate the integrand $\\max_tN_I(t, \\alpha)p(\\alpha)$. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Define user function $\\max_t N_I(t, \\alpha)p(\\alpha)$ and integral bounds \n",
+    "\n",
+    "We call the integrand $\\max_t N_I(t, \\alpha)p(\\alpha)$ `f_height_of_peak_weighted`  since it computes the height of the infection peak weighted with the prior on $\\alpha$. One call of `f_height_of_peak_weighted` will run `num_gil` Gillespie runs. The integral bounds have units $\\alpha$, i.e., units of infection time divided by recovery time; they define the domain $\\Omega = [lower\\_bounds, upper\\_bound]$. We choose `80` to be the upper bound since we assume that the infection rate will not be larger than 80 times the recovery rate.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define the integrand \n",
+    "f_height_of_peak_weighted = seir.height_of_peak_weighted(meanmax)\n",
+    "\n",
+    "# define integral bounds\n",
+    "lb = 1. # lower integral bound\n",
+    "ub = 80. # upper integral bound"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Get initial observations\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The quadrature model needs a few random staring locations to get an initial calibration of its hyperparameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# number of initial points\n",
+    "N_init_task1 = 3\n",
+    "\n",
+    "# inital traning points: locations X_init sampled uniformly from [lower_bound, upper_bound]. \n",
+    "X_init_task1 = np.random.rand(N_init_task1, 1)*(ub-lb) + lb \n",
+    "Y_init_task1 = f_height_of_peak_weighted(X_init_task1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create Emukit surrogate model (the emulator) for the integrand function\n",
+    "\n",
+    "We use a GPy model wrapper which is already in Emukit. Then we choose the vanilla Bayesian quadrature method and create the loop. We then run the loop for a certain number of iterations. Finally we integrate the model and collect the estimated expectation of the height of the infection peak and its variance."
```

Comment:
"We use GPy package to build the model and  GPy model wrapper ..."

Reply:
I'll rephrase